source/js/buyers-guide/product-analytics.js

```diff
@@ -0,0 +1,92 @@
+import ReactGA from 'react-ga';
+
+function getElementGAInformation(pageTitle, productName) {
+  return {
+    'product-live-chat`': {
+      category: `product`,
+      action: `customer support link tap`,
+      label: `support link for ${productName}`
+    },
+    'donate-button': {
+      category: `site`,
+      action: `donate tap`,
+      label: `${pageTitle} donate header`
+    },
+    'nav-social-button-fb': {
+      category: `site`,
+      action: `share tap`,
+      label: `share site to Facebook`,
+      transport: `beacon`
+    },
+    'nav-social-button-twitter': {
+      category: `site`,
+      action: `share tap`,
+      label: `share site to Twitter`,
+      transport: `beacon`
+    },
+    'nav-social-button-link': {
+      category: `site`,
+      action: `copy link tap`,
+      label: `copy link to site `
+    },
+    'nav-social-button-email': {
+      category: `site`,
+      action: `share tap`,
+      label: `share site to Email`,
+      transport: `beacon`
+    },
+    'product-copy-link-button': {
+      category: `product`,
+      action: `copy link tap`,
+      label: `copy link ${productName}`
+    },
+    'creep-vote': {
+      category: `product`,
+      action: `opinion submitted`,
+      label: `opinion on ${productName}`
+    },
```

Comment:
- panning Creep-O-Meter seems to trigger this event as well. I'm not sure if this is the desired event we wanna attach to this action
<img width="527" alt="image" src="https://user-images.githubusercontent.com/2896608/47325399-619c6a00-d618-11e8-9c21-daa133e453bc.png">

- answering "How likely are you to buy it?" double trigger this event
<img width="529" alt="image" src="https://user-images.githubusercontent.com/2896608/47325434-8db7eb00-d618-11e8-8e6d-71ce09f33b43.png">

Reply:
looks like this was tacked onto "the form" which means everytime you click the form it fires! adding an `id` to the vote button and fixing it to use that instead. Good catch!